f3sim.go

```diff
@@ -34,10 +34,10 @@ func main() {
 			LatencyMean: *latencyMean,
 		}
 		graniteConfig := f3.GraniteConfig{
-			Delta:          *graniteDelta,
+			Delta:          time.Duration(*graniteDelta),
```

Comment:
These all need to be multiplied by `time.Second`. I'm actually a bit confused why this type-checked ok at all